service-jee-jaxrs/README.md

```diff
@@ -64,6 +64,8 @@ different host. To do this edit `keycloak.json` and add:
 }
 ````
 
+As an alternative you can create the client by importing the file [client-import.json](config/client-import.json) and
+copying [config/keycloak-example.json](config/keycloak-example.json) to `config/keycloak.json`.
```

Comment:
As an alternative, you can create the client by importing the [client-import.json](config/client-import.json) file and
copying [config/keycloak-example.json](config/keycloak-example.json) to `config/keycloak.json`.